### PR TITLE
Empty badge version

### PR DESCRIPTION
**Pull request by EmelieLitwin**
**Describe pull-request**  
Added a prop for small size (simple red dot without number) and one for visibility.

**Solving issue**  
Github: [#265](https://github.com/scania-digital-design-system/sdds/issues/265)
Azure: [AB#1337](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1337)

### CODE REVIEW

**Review by albinscania — APPROVED** [review]
Looks good! Nothing that you added that I think needs to change. (there is this comment about parsing the number, but that was there from before, but I would remove it in the future :)

**Comment by alexlioliopoulos** [comment]
Nice! My only comment is regarding the fact that we have the value field in the small variant.

image

which is not applicable in that story, right?

File: components/src/components/badges/badges-component.tsx [hunk 1]
```diff
@@ -6,43 +6,85 @@ import { Component, h, Prop, Watch, State } from '@stencil/core';
   shadow: true,
 })
 export class SddsBadges {
+  /**Value shown in badge*/
   @Prop() value: string = '';
 
+  /**Changes visibility of badge*/
```
Comment (theJohnnyMe) [inline components/src/components/badges/badges-component.tsx:12]: Super nice with using comments ;)

**Review by theJohnnyMe — APPROVED** [review]
NIce work Emelie ;)

**Pull request merged**